data/presets/leisure/track/athletics/long_jump.json

```diff
@@ -0,0 +1,18 @@
+{
+    "icon": "fas-running",
+    "geometry": [
+        "point",
+        "line",
+        "area"
+    ],
+    "tags": {
+        "leisure": "track",
+        "sport": "athletics",
+        "athletics": "long_jump"
+    },
+    "reference": {
+        "key": "athletics",
+        "value": "long_jump"
+    },
+    "name": "Long Jump Runway"
```

Comment:
Users less familiar with track and field may search for “(discipline) track” instead of “(discipline) runway”; consider adding an alias for that terminology.